src/main/groovy/com/rjrudin/marklogic/gradle/MarkLogicPlugin.groovy

```diff
@@ -426,4 +422,8 @@ class MarkLogicPlugin implements Plugin<Project> {
         deployer.setCommands(commands)
         return deployer
     }
+    
+    def host(project) { 
+      project.hasProperty("mlHost") ? project.property("mlHost") : project.property("marklogic.host")

```

Comment:
Test out what happens if neither mlHost nor marklogic.host is present - project.property may throw an error if the property doesn't exist.

For testing, I have ml-gradle and the gh-pages branch of marklogic-java in the same "workspace" directory. I change ml-gradle, then I do something like "gradle -Pversion=something publish". Then I go to examples/sample-project and modify its gradle.properties file to depend on version "something" of ml-gradle. Then to test this, you could just run "gradle -i tasks" to see what ml-gradle prints out for the property values.